js/msp.js

```diff
@@ -63,7 +63,7 @@ var MSP_codes = {
     MSP_SET_SPECIAL_PARAMETERS: 99,
 
     // Multiwii MSP commands
-    MSP_IDENT:              100,
+    //MSP_IDENT:              100,
```

Comment:
You mean to leave the uncommented ? I was thinking back and forth for this but I saw some other commands, e.g.[ MSP_BIND on row 113](https://github.com/betaflight/betaflight-configurator/blob/master/js/msp.js#L113) so I did the same for this. This will awareness of the commands but still show that they are unused in this app.